SamplifyAPIClient/SamplifyAPIClient.csproj

```diff
@@ -21,5 +21,6 @@
   <ItemGroup>
     <Compile Remove="Event.txt" />
     <Compile Remove="Order.cs" />
+    <Compile Remove="EmptyClassM.cs" />
```

Comment:
Excluding a file from project configures these <Compile Remove> tags. This is fine for a WIP project but if the files are no longer part of the lib. we need to remove these, rebuild (to make sure everything is fine) and then push the code.